### ISSUE TITLE
FAST model pre-training single-arm and dual-arm data

### ISSUE THREAD

**Issue by newer79** [Physical-Intelligence/openpi#836]
Hello, I would like to inquire about the details of the pre-training of PI0.5.
When a single arm is filled to the specified dimension, will the loss calculation during training mask out the filled action dimensions? Since the original actions have become variable-length actions after being processed by the FAST tokenizer, and it is impossible to distinguish between the filled and non-filled parts. Or will it not be masked, but all action dimensions will be involved in the loss calculation?
When making a deduction, is the action dimension after the FAST decode the maximum action dimension? Then, should we select the corresponding action dimension based on whether it is a single arm or a double arm? Or will the FAST decode automatically revert to the corresponding single-arm or double-arm dimension?
Thanks.
